packages/textarea/test/lion-textarea-integrations.test.js

```diff
@@ -1,11 +1,34 @@
-import { runFormatMixinSuite } from '@lion/form-core/test-suites';
+import { defineCE } from '@open-wc/testing';
+import {
+  runInteractionStateMixinSuite,
+  runFormatMixinSuite,
+  runNativeTextFieldMixinSuite,
+} from '@lion/form-core/test-suites';
 
-import '@lion/textarea/define';
+import { LionTextarea } from '../src/LionTextarea.js';
```

Comment:
Is there a reason to not use textarea/define anymore for this test?

Reply:
It didn't work before, but now it works again. Thanks for letting me double check it :)